src/cli.cr

```diff
@@ -211,12 +211,17 @@ module DppmRestApi::CLI
     )
   end
 
+  # Require ALL variables
+  private macro require_all_args
+    required {{@def.args.splat}}
```

Comment:
This is not really the best away to do this.
If the variables aren't optional, positional CLI arguments (`arguments`) can be used.
This isn't on this PR scope, I can submit one after.

Reply:
Relevant changes have been reverted.